site/_docs/reference.md

```diff
@@ -1992,6 +1993,29 @@ Note:
 | value IS JSON ARRAY                               | Whether *value* is a json array, *value* is in character string type
 | value IS NOT JSON ARRAY                           | Whether *value* is not a json array, *value* is in character string type
 
+#### MySQL Specific Operators
+
+| Operator syntax                                   | Description
+|:------------------------------------------------- |:-----------
+| JSON_TYPE(value) | Returns a string indicating the type of a JSON **value**. This can be an object, an array, or a scalar type
+
+Example SQL:
+
+```SQl
```

Comment:
Could you please fix typo: `SQl` -> `SQL`

Reply:
Thank you very much for helping me review again and again.I have already change it.